js/dropdown/shared/Dropdown.svelte

```diff
@@ -18,6 +18,23 @@
 	export let show_label: boolean;
 	export let container = true;
 	export let allow_custom_value = false;
+	export let filterable = true;
+
+	let filter_input: HTMLElement;
+
+	let show_options = false;
```

Comment:
All these variables are being redefined below. I think we can get rid of them?

Reply:
nice catch thanks